recipes/freecad/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "freecad" %}
+{% set version = "0.18b0" %}
+{% set sha256 = "1462df34caa67791bb7512f6c464f16a15d49f0d9f777996cb4e7b8174c57191" %}
+{% set commit = "e49a5af3ae2f976017f1156bdc113594f81d3fb5" %}
+
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}_{{ version }}.tar.gz
+    url: https://github.com/FreeCAD/FreeCAD/archive/{{ commit }}.tar.gz
```

Comment:
Is there a released version that we can package?

Reply:
yes, but the latest version is most compatible with py3. using any older release doesn't make much sense.

Reply:
AFAICT, @scopatz, this seems to be the remaining issue.